specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2020-03-01/cosmos-db.json

```diff
@@ -0,0 +1,7248 @@
+    "description": "Azure Cosmos DB Database Service Resource Provider REST API",
+    "version": "2020-03-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "Impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}": {
+      "get": {
+        "operationId": "DatabaseAccounts_Get",
+        "x-ms-examples": {
+          "CosmosDBDatabaseAccountGet": {
+            "$ref": "./examples/CosmosDBDatabaseAccountGet.json"
+          }
+        },
+        "description": "Retrieves the properties of an existing Azure Cosmos DB database account.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/accountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The database account properties were retrieved successfully.",
+            "schema": {
+              "$ref": "#/definitions/DatabaseAccountGetResults"
+            }
+          }
+        }
+      },
+      "patch": {
+        "operationId": "DatabaseAccounts_Update",
+        "x-ms-examples": {
+          "CosmosDBDatabaseAccountPatch": {
+            "$ref": "./examples/CosmosDBDatabaseAccountPatch.json"
+          }
+        },
+        "description": "Updates the properties of an existing Azure Cosmos DB database account.",
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/accountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "name": "updateParameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/DatabaseAccountUpdateParameters"
+            },
+            "description": "The parameters to provide for the current database account."
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
For long running operation, will you also add the response code of 201, 202?

Reply:
The previous versions of this API return 200. It hasnt changed with this version update.

Reply:
I am trying to understand the behavior of your service. So for this API, will your service only return the response code of 200 only?

Reply:
Yes, it does only return 200